.github/workflows/push.yml

```diff
@@ -93,7 +93,14 @@ jobs:
         run: export LATEST_TAG=$(git describe --tags --abbrev=0)
       - name: Echo git tag
         run: echo $LATEST_TAG
-      - run: sbt test ciReleaseTagNextVersion ciReleaseSonatype
+      - name: Decrypt private-key.pem
+        run: openssl aes-256-cbc -K ${{ env.KEY_663dec309dc8 }} -iv ${{ env.IV_663dec309dc8 }} -in private-key.pem.enc -out private-key.pem -d
+      - name: Import GPG key
+        run:  gpg --passphrase ${{ secrets.PGP_PASSPHRASE }} --batch --import private-key.pem
+      - name: Export secret key
+        run: gpg --passphrase ${{ secrets.PGP_PASSPHRASE }} --batch --export-secret-key -a CHANGE_TO_KEY_ID > ~/.sbt/gpg/secring.asc
+      - name: Release to Sonatype
+        run: sbt ciReleaseTagNextVersion ciReleaseSonatype
```

Comment:
missing `test` step?

Reply:
Nope @hubertp. The `release` job depends on `test-codepropertygraph`, which is where `sbt test` runs. Found it to be cleaner that way